media-video/libav/libav-11.9999.ebuild

```diff
@@ -134,8 +135,9 @@ REQUIRED_USE="
 	amr? ( gpl ) aac? ( gpl ) x264? ( gpl ) X? ( gpl ) cdio? ( gpl ) x265? ( gpl )
 	test? ( encode zlib )
 	fontconfig? ( truetype )
+	?? ( gnutls libressl )
```

Comment:
alternative "patch" is already at bugzilla. I'll commit this with maintainer timeout now anyway